### ISSUE TITLE
Wrong From name shown in the message

### ISSUE THREAD

**Issue by shashidharg** [protz/thunderbird-conversations#370]
Hello,

First of all let me say your addon for Thunder bird is very cool :)
However I just noticed a strange bug. We get email alerts from bugzilla tool, and when all messages are collected up it shows the wrong From name in the message

First email would be From : Bugzilla (for xyz) bugs@work.com when the bug is created

When the bug is updated by another person we get the email alert, From: Bugzilla (for abc) bugs@work.com and so on. 

so as u see the email address is same but the From Name is different every time we get the emails. The issue is that when the emails are grouped into a GMail view. All of them show : Bugzilla (for xyz) 

In other words we will have a lot of email with same email id but different From name. This is causing all the email message to have the From name of the first person and not the one that is related to that particular email.

Hope I am clear

Regards,
Shashidhar.G
